AEPPlaces/Sources/Places+EdgeEvents.swift

```diff
@@ -0,0 +1,100 @@
+/*
+ Copyright 2023 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPCore
+import AEPServices
+import Foundation
+
+extension Places {
+    
+    static let MASK_EVENT_TYPE_PATH = "xdm.eventType"
+    static let MASK_POIID_PATH = "xdm.placeContext.POIinteraction.poiDetail.poiID"
```

Comment:
nit: would it make sense to add these mask constants to the PlacesConstants file? i see these strings used in the places tests as well.